CarRentalService/Domain/BodyType.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace CarRentalService.Domain.Domain;
+/// <summary>
+/// Body types for vehicles
+/// </summary>
+public enum BodyType
+{
+    /// <summary>
+    /// Sedan body type
+    /// </summary>
+    Sedan,
+
+    /// <summary>
+    /// Hatchback body type
+    /// </summary>
+    Hatchback,
+
+    /// <summary>
+    /// SUV - sport utility vehicle
+    /// </summary>
+    SUV,
```

Comment:
В C# акронимы также именуются в PascalCase.
```suggestion
    Suv,
```